### DISCUSSION TITLE
Weird

### DISCUSSION THREAD

**Discussion by MvPGEO** [Sidekick-Poe/Sidekick#409] [General]
Why you cant select folder of installation? It always installs into C:\Users\user\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Sidekick .... I want custom installation.

**Comment by domialex** [comment]
You can use the portable version, there's a button on the website and it's available here:
https://github.com/Sidekick-Poe/Sidekick/releases

![image](https://github.com/user-attachments/assets/82cd7054-6ce9-4a76-b7e9-409e9c7cbc40)
